doc/api/tty.md

```diff
@@ -88,6 +88,17 @@ process.stdout.on('resize', () => {
 });
 ```
 
+*Note:* On Windows resize events will be emitted only if stdin is unpaused 
```

Comment:
Hmm...I copied this mistake from [async_notes.md](https://github.com/nodejs/node/blob/master/doc/api/async_hooks.md) Maybe worth a separate PR?